sw.js

```
@@ -1,5 +1,5 @@
 // Service Worker for ESP32Tool PWA
-const CACHE_NAME = 'esp32tool-v1.6.7';
+const CACHE_NAME = 'esp32tool-v1.7.0';
```

ESLint

[error] Replace `'esp32tool-v1.7.0'` with `"esp32tool-v1.7.0"`

(prettier/prettier)